CMakeLists.txt

```diff
@@ -123,7 +123,7 @@ if(ENABLE_OPENSSL OR ENABLE_PICOTLS)
   find_package(OpenSSL 1.1.1 REQUIRED)
 endif()
 if(ENABLE_WOLFSSL)
-  find_package(wolfssl 5.5.0 REQUIRED)
+  find_package(wolfssl 5.5.0 CONFIG REQUIRED)
```

Comment:
wolfSSL CMake support lags behind its `./configure` AFAICT.
Does this change let the latter continue to work as it was before? If not, I'd consider leaving it as-is.

Reply:
no it doesn't automatically falls back to pkg-config. I have pushed update to Findwolfssl module which first tries finding CMake export config from openssl and falls back to pkg-config, trying to replicate it.

Reply:
Shouldn't tweaking wolfSSL Find module be a separate PR?

Is this required to export the wolfSSL crypto lib via ngtcp2's config?

Reply:
> Is this required to export the wolfSSL crypto lib via ngtcp2's config?

yes it is required, because exported config needs to be useful regardless of how ngtcp2's dependencies were built.  When generating export config, CMake propagates all exported targets dependencies, so that consumers of ngtcp2_crypto_wolfssl know that wolfssl needs to be linked too. For that export to work, Findwolfssl should create targets which looks the same, regardless of how they were found.